CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Chores
 
+- Fixed the properties and providers for the Send stories ([PR 3047](https://github.com/input-output-hk/daedalus/pull/3047))
```

Comment:
The new entry should be under `vNext`
Please refer to https://github.com/input-output-hk/daedalus/pull/3042#discussion_r996721125